### ISSUE TITLE
`clarinet test` exit code

### ISSUE THREAD

**Issue by whoabuddy** [hirosystems/clarinet#695]
In the Clarinet channel of the Stacks Discord someone brought up having to scan the output of `clarinet test` in order to have the action fail at the test step.

I couldn't remember if Clarinet used to or should exit with status 1 when a test fails, but [the third example in this old comment](https://github.com/hirosystems/clarinet/issues/39#issuecomment-858619593) makes me think it used to work that way.

Using Clarinet locally I can confirm `clarinet test` exits with status 0 when tests fail.

result: `FAILED | 67 passed | 6 failed (689ms)`
```bash
$ echo $?
0
```

Also seen in [this GH action](https://github.com/citycoins/protocol/actions/runs/3536024922/jobs/5934677571) where the step passes when IMO it shouldn't.

**Comment by lgalabru** [comment]
Good catch thanks @whoabuddy! There was indeed a regression - to be addressed with incoming release `v1.2.0`.

**Comment by ummarikram** [comment]
Hi, the latest version `v1.2.0` is not able to execute the pulled docker image properly. It throws an error of ```error "/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by clarinet)```

**Comment by lgalabru** [comment]
Thank you for signaling this problem @ummarikram, I created a dedicated issue here https://github.com/hirosystems/clarinet/issues/697.

**Comment by whoabuddy** [comment]
Confirmed, thank you!
